controllers/tmf-apis/usageManagement.js

```diff
@@ -0,0 +1,104 @@
+var AccountingService = require('./../../db/schemas/accountingService'),
+    async = require('async'),
+    storeClient = require('./../../lib/store').storeClient,
+    utils = require('./../../lib/utils'),
+    tmfUtils = require('./../../lib/tmfUtils');
+
+var usageManagement = ( function () {
+
+    /**
+     * Check if the API Key passed is a valid API Key
+     *
+     * @param  {Object}   req      Incoming request.
+     */
+    var validateApiKey = function (req, callback) {
+
+        var apiKey = req.get('X-API-KEY');
+
+        if (!apiKey) {
+
+            return callback({
+                status: 401,
+                message: 'Missing header "X-API-KEY"'
+            });
+
+        } else {
+
+            AccountingService.findOne( {apiKey: apiKey}, function (err, result) {
+
```

Comment:
I prefer having a single return, you can save the response in a var, and then execute a return callback after the if statement